src/apps/material/material.tsx

```diff
@@ -101,9 +103,9 @@ const Material: React.FC<MaterialProps> = ({ wid }) => {
     t
   });
 
-  if (!currentManifestation) {
-    return null;
-  }
+  const infomediaId = currentManifestation.access.find(
+    (item) => item.__typename === "InfomediaService"
+  ) as InfomediaService;
```

Comment:
Please rename the variable `infomediaId`. I suggest just calling it `infomedia` or `infomediaAccess`.

The variable does not (only) represent an id. Accessing via `infomediaId.id` (double id) looks odd.

However I do think this is an example where it is OK to use `as`.